### ISSUE TITLE
input format

### ISSUE THREAD

**Issue by emiramaro** [mirthAI/Med3DVLM#2]
Dear authors, 

Thank you very much for the great work!

I'm very interested to incorporate your model in my current work. 

I saw that some of the demo data are either ni.gz files that separated into multiple axes or one whole nii file. I wonder what would be the best approach to fine-tune the model to a new dataset? Separate the nii files into multiple files or just ingest the whole nii file? 

Thank you!

**Comment by Magic-Xin** [comment]
Thank you for your interest and support!
We used separated axes data because they come from the M3D-Cap dataset, and the demo samples were randomly selected from the test set. If your fine-tuning task does not require the model to recognize the imaging plane, you can directly use the whole NIfTI files without separating them.

**Comment by emiramaro** [comment]
Got it, thank you very much for your response!

One question, I didn't see the 'vqa_finetune.sh' script on the repo for the VLM finetuning, is it modified as the 'finetune_lora.sh'?

**Comment by emiramaro** [comment]
Thank you very much for the clarification!

I'm currently trying to finetune the model, but just realized that the pretrained multimodal projection weights are not included in the hugging face repo. 

Did I miss it somewhere?

**Comment by Magic-Xin** [comment]
We have bundled all the weights into the Med3DVLM-Qwen-2.5-7B model on Hugging Face, including the vision encoder, multimodal projector, the original Qwen 2.5-7B weights, and the LoRA adapters.
If you need to use a specific module's weights separately, you can extract them from the full model. You can refer to the `model.safetensors.index.json` file in the Hugging Face repo to see the mapping between modules and weights.
